eng/tools/RepoTasks/RepoTasks.csproj

```diff
@@ -20,9 +20,9 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)'">
-    <PackageReference Include="Microsoft.Build.Framework" Version="16.3.0" />
-    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="16.3.0" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.3.0" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="16.9.0" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="16.9.0" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.9.0" />
```

Comment:
Given errors in CI build and much lower required version in global.json, these might need further changes to get working. If that's possible, please update https://github.com/dotnet/aspnetcore/blob/main/global.json#L22

Reply:
Won't that update the required VS version for contributors? Do we want to bump that all the way up to 16.9? I think 16.6 would be recent enough to know about the new TFMs

Reply:
Yes, it'll update what contributors need. Since 16.9 is the current public non-preview build of VS, I'm not positive that's a show-stopper.

But, if we decide not to jump to 16.9, I'd prefer we moved to 16.8 instead of 16.6 because it includes an `msbuild` supporting the new functions for the new TFMs.

Reply:
I'll go for 16.8